examples/mac.rs

```diff
@@ -0,0 +1,25 @@
+
+extern crate notify_rust;
+
+#[cfg(target_os = "macos")]
+fn main() -> Result<(), Box<std::error::Error>> {
+    use notify_rust::{Notification, get_bundle_identifier_or_default, set_application};
+
+    let safari_id = get_bundle_identifier_or_default("Safari");
+    set_application(&safari_id)?;
+    set_application(&safari_id)?;
```

Comment:
Why is this method called twice?
If there is a specific reason, a comment would help.